cl/scrapers/management/commands/cl_scrape_oral_arguments.py

```diff
@@ -173,7 +173,8 @@ def scrape_court(
                     backscrape=backscrape,
                 )
                 process_audio_file.apply_async(
-                    (audio_file.pk,), countdown=random.randint(0, 3600)
+                    args=(audio_file.pk,),
+                    countdown=random.randint(0, 3600),
```

Comment:
I'm glad you tweaked the formatting here, so I could see this. The jitter we put here was to help with our old architecture so that a bunch of new audio files wouldn't clog up the server. We're more scalable now, so I'd say we should take the random jitter out of here and we should be fine. 

Otherwise, some RT alerts will be delayed by an hour, which isn't great.

Reply:
Great, I've removed the jitter from this task so now is just: `process_audio_file.delay(audio_file.pk)`